roboteam_ai/src/control/ballHandling/BallHandlePosControl.cpp

```diff
@@ -298,7 +321,18 @@ RobotCommand BallHandlePosControl::goToIdleBall(
     else {
         numTreesTarget = ball->pos + ballToTarget.stretchToLength(maxBallDistance);
     }
-    return NumTreePosControl::getRobotCommand(robot, numTreesTarget);
+    auto robotCommand = NumTreePosControl::getRobotCommand(robot, numTreesTarget);
```

Comment:
Take posController from robot

Reply:
You did not fix this.

Reply:
see the above comments

Reply:
this is a call to the baseclass and is thus happening inside the robot object ;-) 

Although the name is confusing. A comment would be great.

Reply:
(the weird notation is because the function is overridden in ballhandle)